pkgs/by-name/ci/cinny-desktop/package.nix

```diff
@@ -14,26 +13,30 @@
   dbus,
   glib,
   glib-networking,
-  libayatana-appindicator,
   webkitgtk_4_0,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "cinny-desktop";
   # We have to be using the same version as cinny-web or this isn't going to work.
-  version = "4.2.3";
+  version = "4.3.0";
```

Comment:
Update to [v4.3.1](https://github.com/cinnyapp/cinny-desktop/releases/tag/v4.3.1)